.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,12 @@
+settings:
+  jira_project_key: MIRENG
+  status_mapping:
+    opened: Triaged
```

Comment:
Do we really want this? I'd expect "Untriaged" as per the linked README

Reply:
Since this will only act on issues that we put a label on (I said "triaged"), I thought maybe we don't need this extra step.

Can be convinced otherwise, but we'd need a different label to key on.

Reply:
OK, I thought it would "triage" everything that was open

Reply:
I'll add some comments on the config to clarify

Reply:
Done.

Reply:
OTOH, we have many Github labels I wouldn't consider "Triaged", E.g. "wontfix".

Reply:
Right, but do we want/need to add them to JIRA? I had this as "we want to work on this soon, add it to JIRA so we can plan it in".